src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -135,10 +135,10 @@ class MulticopterPositionControl : public ModuleBase<MulticopterPositionControl>
 	vehicle_land_detected_s _vehicle_land_detected = {
 		.timestamp = 0,
 		.alt_max = -1.0f,
-		.landed = true,
 		.freefall = false,
-		.ground_contact = false,
-		.maybe_landed = false,
+		.ground_contact = true,
+		.maybe_landed = true,
+		.landed = true,
```

Comment:
Same here as previous comment above.  I think these new values are the correct states to be initialized to.  This looks good to me!

Reply:
Here the initialization is useful since if there was no land detected published the position controller will assume the vehicle is landed and before only one flag was able to be true so it was `landed`, now it makes sense that `landed` implies `maybe_landed` and `ground_contact`.